BeatSaberMultiplayer/BeatSaberMultiplayer.csproj

```diff
@@ -72,7 +72,7 @@
     <Reference Include="System.Drawing" />
     <Reference Include="System.IO.Compression, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\..\..\..\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.netcore.app\2.0.0\ref\netcoreapp2.0\System.IO.Compression.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\Program Files (x86)\Steam\steamapps\common\Beat Saber\Beat Saber_Data\Managed\System.IO.Compression.dll</HintPath>
```

Comment:
This isn't related to beat saber 0.13.1, but it improves compatibility for building on different systems I think. The original didn't work on mine. If this version doesn't work for your system, I'll revert it.